src/overlays/actors/ovl_Obj_Ice_Poly/z_obj_ice_poly.c

```diff
@@ -13,9 +13,13 @@
 void ObjIcePoly_Init(Actor* thisx, GlobalContext* globalCtx);
 void ObjIcePoly_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ObjIcePoly_Update(Actor* thisx, GlobalContext* globalCtx);
-void ObjIcePoly_Draw(Actor* thisx, GlobalContext* globalCtx);
+void ObjIcePoly_Draw(ObjIcePoly* this, GlobalContext* globalCtx);
```

Comment:
I suppose this isn't intentional?
```suggestion
void ObjIcePoly_Draw(Actor* thisx, GlobalContext* globalCtx);
```